config/webpack.config.js

```diff
@@ -200,6 +200,11 @@ module.exports = function (
 		return Array.isArray(paths) ? paths : [paths];
 	};
 
+	const mainEntry = app?.entry?.main || app.context;
+	if (app?.entry?.main !== undefined) {
+		delete app.entry.main;
+	}
+
```

Comment:
How about
```suggestion
	const {mainEntry, ...restEntries} = app.entry;
```

Reply:
"main" is not guaranteed to be at the beginning of the array.